openedx/features/content_type_gating/partitions.py

```diff
@@ -147,6 +152,21 @@ def get_group_for_user(cls, course_key, user, user_partition, **kwargs):  # pyli
         if not ContentTypeGatingConfig.enabled_for_enrollment(user=user, course_key=course_key):
             return cls.FULL_ACCESS
 
+        # TODO: clean up as part of REV-100
+        experiment_data_holdback_key = EXPERIMENT_DATA_HOLDBACK_KEY.format(user)
```

Comment:
It occurs to me that maybe now that we have the config model, this should be part of `enabled_for_enrollment` on each of the models? Just to make sure that it's checked correctly in all of the places where we're checking enabledment for a specific user/enrollment?